CollectionActions.rb

```diff
@@ -129,6 +131,7 @@ def store_items(obj_item, location: nil)
     end
   end
 
+  # NOTE: is this method meant to tell someone to trash an item or is it to make a table of items with the place to trash them? I am confused as it has a show block, but it returns a table? 
```

Comment:
As the note says -- is this a method to give instructions to the technician or to make a table? It has a show block, but then it returns a table.

Reply:
It is to give instructions.   It wasn't intended to return a table.  I think thats just from the inherent ruby thing where it returns what ever object is the last line in the method...    Not sure, not supposed to return anything just display the table with instructions to throw stuff away.